docs/modules/deployment/pages/upgrade/restore/centos-rhel8/restore-centos-rhel8.adoc

```diff
@@ -30,7 +30,7 @@ sudo cp /opt/opennms/etc/opennms.service /lib/systemd/system/
 
 ._Assign CAP_NET_RAW capabilities_
 
-See xref:deployment:core/centos-rhel/initialize-core.adoc#initialize-core-cap[Assign CAP_NET_RAW capabilities].
+See xref:core/rpm/initialize-core.adoc#initialize-core-cap[Assign CAP_NET_RAW capabilities].
```

Comment:
We ship with this enabled in the systemd unit.  I think this callout can be removed entirely.